src/plugins/telemetry/telemetry_impl.cpp

```diff
@@ -1076,7 +1076,13 @@ void TelemetryImpl::process_battery_status(const mavlink_message_t& message)
 
     Telemetry::Battery new_battery;
     new_battery.id = bat_status.id;
-    new_battery.voltage_v = bat_status.voltages[0] * 1e-3f;
+    new_battery.voltage_v = 0.0f;
+    for (int i = 0; i < 10; i++)
```

Comment:
Nitpick: since the battery id is a `uint8_t`, it means that MAVLink supports up to 255 batteries, right?

Wouldn't it make sense to go up to 255 instead of 10 here? I know it's highly unlikely that it ever goes higher than 10, but with 255 we just follow the MAVLink specs. And because we `break` after the first `UINT16_MAX`, it doesn't add any override.

What do you think?

Reply:
That is a valid suggestion Jonas.  It is always good to stay within the specs.  It will also make the code more future proof.